modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -2262,3 +2262,24 @@ def func(df):
         return df_copy
 
     eval_general(modin_df, pandas_df, func)
+
+
+def test_index_of_empty_frame():
+    # Test on an empty frame created by user
+    md_df, pd_df = create_test_dfs(
+        {}, index=pandas.Index([], name="index name"), columns=["a", "b"]
+    )
+    assert md_df.empty and md_df.empty
```

Comment:
uh? why double-check `md_df.empty`?.. should the second one be `pd_df.empty`?

Reply:
omg, copy-paste is evil...

fixed it